development/run-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+./vendor/bin/phpunit --testdox
+./vendor/bin/behat
+dredd
+
+export TOKEN=3a4f118c7a927a1e11c2eea2818025ee096ca56f
+
+curl -X PUT http://nginx/list/default/870970-basis:50936155 -H "Authorization: Bearer ${TOKEN}"
+
+curl -X GET http://nginx/list/default  -H "Authorization: Bearer ${TOKEN}"
```

Comment:
I wonder what is going on here?

Normally I would not consider it a good idea to include such a token in VCS.

In general testing of the API should be covered by Dredd and Behat.

Reply:
Yeah was just using livrem and seler.
Removing that part
Token was not a real token just a random string